src/components/Loader/index.js

```diff
@@ -14,6 +14,7 @@ export default class Loader extends React.Component {
 
   state = {
     hasError: false,
+    error: null,
```

Comment:
Since we have `error` field in `state`, do we still need `hasError` as a boolean flag?

Reply:
We should be able to remove it since it's redundant. Good catch!